enterprise/com/src/main/java/org/neo4j/com/storecopy/StoreCopyClient.java

```diff
@@ -333,6 +339,7 @@ private GraphDatabaseService newTempDatabase( File tempStore )
                 .setConfig( "dbms.backup.enabled", Settings.FALSE )
                 .setConfig( GraphDatabaseSettings.logs_directory, tempStore.getAbsolutePath() )
                 .setConfig( GraphDatabaseSettings.keep_logical_logs, Settings.TRUE )
+                .setConfig( GraphDatabaseSettings.logical_logs_location, tempStore.getAbsolutePath() )
```

Comment:
What's the incentive to explicitly set the `logical_logs_location` to the store dir? (a) It's the default already and (b) why override it?

Reply:
Default value is based on "database_path" which is based on data_directory that is based on neo4j_home. And they are not defined for temp database and they don't need to be defined there also since in temp db we have a flat structure where we just put the store and tx logs into a directory without setting up data directory, db name directory, etc.
That's why we overwrite it with exact location of temp store directory